### PR TITLE
Kubernetes模块使用Bridgx原生API

### PR DESCRIPTION
**Pull request by weenxin**
Pull Request Check List:
- [x] rebase origin/dev branch (**passed**)
- [x] make format & make build  (**passed**)
- [x] union test or test in dev env(**passed**)

Detail in branch:
- [ ]
- [ ]

### CODE REVIEW

File: internal/service/gf_cluster.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	_, instances, total, err := GetInstancesByAccounts(ctx, accountKeys, []string{"running"}, 1, 10, "", "", clusterName)
+	if err != nil {
+		return nil, 0, err
+	}
+	return instances, int(total), err
+
+}
+
+//GetClusterAKSK 根据集群获取aksk
+func GetClusterAKSK(ctx *gin.Context, clusterName string) (*model.Account, error) {
```
Comment (liqunn) [inline internal/service/gf_cluster.go:110]: naming `GetClusterAKSK ` 感觉GetClusterAccount好一些？返回的是*model.Account

File: internal/service/gf_cluster.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	return instances, nil
+}
+
+//getClusterInstances 根据peage获取实例列表
+func getClusterInstances(ctx context.Context, user *authorization.CustomClaims, clusterName string, pageNum, pageSize int) ([]model.Instance, int, error) {
+	accountKeys, err := GetAksByOrgAkProvider(ctx, user.OrgId, "", "")
+	if err != nil {
+		return nil, 0, err
+	}
+	_, instances, total, err := GetInstancesByAccounts(ctx, accountKeys, []string{"running"}, 1, 10, "", "", clusterName)
```
Comment (liqunn) [inline internal/service/gf_cluster.go:101]: `"running"` 同上

File: cmd/api/handler/gf-cluster/cluster/bridgx.go [hunk 3]
```diff
@@ -3,8 +3,9 @@ package cluster
 import (
 	"net/http"
 
+	"github.com/galaxy-future/BridgX/internal/service"
+
```
Comment (liqunn) [inline cmd/api/handler/gf-cluster/cluster/bridgx.go:7]: 空行

File: internal/service/gf_cluster.go [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+		targetCluster := &gf_cluster.BridgxUnusedCluster{
+			ClusterName: cluster.ClusterName,
+			CloudType:   cluster.Provider,
+			Nodes:       nil,
+		}
+		accountKeys, err := GetAksByOrgAkProvider(ctx, user.OrgId, "", "")
+		if err != nil {
+			return nil, 0, err
+		}
+		_, instances, total, err := GetInstancesByAccounts(ctx, accountKeys, []string{"running"}, 1, 10, "", "", cluster.ClusterName)
```
Comment (liqunn) [inline internal/service/gf_cluster.go:32]: "running" 可以用internal/constants/instance.go里定义的Instance状态

File: internal/service/gf_cluster.go [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+	cluster, err := GetClusterByName(ctx, clusterName)
+	if err != nil {
+		return nil, err
+	}
+	account, err := GetAccount(cluster.Provider, cluster.AccountKey)
+	if account == nil {
+		return nil, fmt.Errorf("account not found")
+	}
+
+	return account, err
```
Comment (galaxy-future-liubo) [inline internal/service/gf_cluster.go:120]: return account, nil

File: internal/service/gf_cluster.go [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+func getClusterInstances(ctx context.Context, user *authorization.CustomClaims, clusterName string, pageNum, pageSize int) ([]model.Instance, int, error) {
+	accountKeys, err := GetAksByOrgAkProvider(ctx, user.OrgId, "", "")
+	if err != nil {
+		return nil, 0, err
+	}
+	_, instances, total, err := GetInstancesByAccounts(ctx, accountKeys, []string{"running"}, 1, 10, "", "", clusterName)
+	if err != nil {
+		return nil, 0, err
+	}
+	return instances, int(total), err
```
Comment (galaxy-future-liubo) [inline internal/service/gf_cluster.go:105]: return instances, int(total), nil

**Review by liqunn — APPROVED** [review]

**Review by galaxy-future-liubo — APPROVED** [review]

**Pull request merged**